Unix/protocol/protocol.c

```diff
@@ -458,7 +458,7 @@ void _ProtocolSocket_Aux_ConnectEvent( _In_ Strand* self_)
        the auxiliary function PROTOCOLSOCKET_STRANDAUX_CONNECTEVENT
        which in turn disables that flag allowing the object to be deleted.
 
-    Unique features and special Behavour:
+    Unique features and special Behaviour:
```

Comment:
I prefer en-US, microsoft tends to be en-US. I'm not actively enforcing en-GB or en-US, but I'll flag once that this is en-GB (and again, this PR is too big -- if someone wants to harmonize, they're free to do that outside of this).